force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,55 @@
+trigger OpportunityTrigger on Opportunity (before insert, before update, after insert, after update, before delete) {
+    switch on Trigger.OperationType {
+        when  BEFORE_DELETE {
+            for(Opportunity deletedOpp: Trigger.old){
+                if(deletedOpp.StageName == 'Closed Won'){
+                    deletedOpp.addError('Cannot delete closed opportunity for a banking account that is won');                 
+                }              
```

Comment:
Is there a way to  throw the error only for industry = banking accounts and not all accounts?